packages/gatsby-tinacms-git/gatsby-node.ts

```diff
@@ -19,6 +19,8 @@ limitations under the License.
 import { router as gitRouter, GitRouterConfig, configureGitRemote } from '@tinacms/api-git'
 
 exports.onCreateDevServer = ({ app }: any, options: GitRouterConfig) => {
-  configureGitRemote(options)
+  if (process.env.TINA_CEE !== undefined) {
```

Comment:
Add comment about why it's a undefined check.
Double check the conventions for that too.